src/NuGet.Core/NuGet.Protocol/RemoteRepositories/RemoteV2FindPackageByIdResource.cs

```diff
@@ -433,27 +434,18 @@ public RemoteV2FindPackageByIdResource(PackageSource packageSource, HttpSource h
             return packageInfos.FirstOrDefault(p => p.Identity.Version == version);
         }
 
-        private Task<IEnumerable<PackageInfo>> EnsurePackagesAsync(
+        private async Task<IEnumerable<PackageInfo>> EnsurePackagesAsync(
```

Comment:
Same as the others, A bunch of these intentionally do not await the task.

Reply:
Done